portfolio/src/main/webapp/script.js

```diff
@@ -12,20 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-// function addRandomGreeting() {
-//   const greetings =
-//       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function getComment() {
+  fetch('/data').then(response => response.json()).then((comment) => {
+      const commentList = document.getElementById('container');
+      commentList.innerHTML = '';
+      for (let i in comment) {
+        commentList.appendChild(WrapWithListFormat(comment[i].message));
+      }
+  });
```

Comment:
```suggestion
  fetch('/data').then(response => response.json()).then((comment) => {
    const commentList = document.getElementById('container');
    commentList.innerHTML = '';
    for (let i in comment) {
      commentList.appendChild(WrapWithListFormat(comment[i].message));
    }
  });
```
https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions
Arrow functions still take +2 indents.